main.js

```diff
@@ -0,0 +1,315 @@
+
+//  jQuery call when a user clicks the search button
+$('.search-button').on('click', function () {
+  var video = $('#video-search').val();
+  // added code to prevent an empty search
+  if(video !== '') {
+    fetch(`https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&q=${video}&type=video&videoEmbeddable=true&key=AIzaSyBG5b1D7dEBXFY0PgaKlMSFBcV3rNA7A8w`).then(function(response) {
+    return response.json();
+  })
+  .then(function(data) {
+  //  function calls to render main video, side videos and also have the side videos "playable"
+  renderMainVideo(data.items[0]);
+  renderSideVideos(data.items);
+  setupReplaceListener();
+  });
+  }
+  });
+  
+
+//  variable to render the main video to the page
+var renderMainVideo = function (mainVideoInfo) {
+  $('.mainVideo').empty();
+  var source = $('#mainvideo-template').html();
+  var template = Handlebars.compile(source);
+  var newHTML = template({mainVideoInfo});
+  $('.mainVideo').append(newHTML);
+};
+
+//  variable to render the five "side" videos
+var renderSideVideos = function (fiveVideoInfo) {
+  $('.sideVideo').empty();
+  var source = $('#fivevideo-template').html();
+  var template = Handlebars.compile(source);
+  for (var i = 0; i < fiveVideoInfo.length; i++) {
+    var newHTML = template(
+      {
+        videoLoop: fiveVideoInfo[i]
+      }
+      );
+    $('.sideVideo').append(newHTML);
+  }
+};
+
+//  variable of apiData for the default video search, which is Disney
+var apiData = {
```

Comment:
I'd expect the default search to be something you pass to the API endpoint, not mock here.